javaSE/src/main/java/com/smartdevicelink/util/HttpRequestTask.java

```diff
@@ -0,0 +1,23 @@
+package com.smartdevicelink.util;
+
```

Comment:
Why do the methods don't have any implementation? I agree that the class should be deprecated. However, it should be an exact copy of the android one for now. Otherwise, it will be a breaking change because the implementation was exposed to the developers who are using JavaSE but now it is not,

Reply:
Methods implemented